pandas/tests/frame/test_reshape.py

```diff
@@ -1273,6 +1273,23 @@ def test_stack_timezone_aware_values():
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize("dropna", [True, False])
+def test_stack_empty_frame(dropna):
+    # GH 36113
+    expected = Series(index=MultiIndex([[], []], [[], []]), dtype=np.float64)
+    result = DataFrame().stack(dropna=dropna)
+    tm.assert_series_equal(result, expected)
+
+
```

Comment:
need a test for the updated error message

Reply:
@jreback test added